### PR TITLE
Follow up questions

### PR DESCRIPTION
**Pull request by allait**
### Remove 'is defined' check from textbox form template

`hidden` question property should only ever be set to `True` if present
(since the only thing it controls is whether the question should be
initially hidden), so we can remove the separate `if hidden is defined`
check, which would cause issues with Nunjucks.

 ### Add followup support for radio and checkbox questions

Merges the boolean and radio question form templates into one and
adds support for the multiple followup questions using the new
`Question.values_followup` property passed in as `followup`.

### CODE REVIEW

**Comment by samuelhwilliams** [comment]
I think this is missing `{% if hidden %} js-hidden related-information{% endif %}` from the fieldset in list-entry.html.

**Comment by allait** [comment]
@samuelhwilliams added hidden flag for list-entry questions
@tombye added some examples to combinations.html
@pcraig3 I've rebased this on top of current master and had to make some more changes to the selection-buttons.html, would appreciate an additional review to check that I haven't missed anything.

File: pages_builder/pages/forms/selection-buttons.yml [hunk 1]
```diff
@@ -252,7 +252,9 @@ examples:
     question: Can you do this essential requirement
     question_number: 93
     value: true
-    followup: requirement-evidence-110
+    followup:
+      true:
+        - requirement-evidence-110
```
Comment (samuelhwilliams) [inline pages_builder/pages/forms/selection-buttons.yml]: This should be the other way around, shouldn't it?

```followup:
  requirement-evidence-110:
    - true```

Comment (allait) [inline pages_builder/pages/forms/selection-buttons.yml]: This is not YAML format used by the content loader, it's the data we'd normally pass into toolkit form macros, so this is what `.values_followup` would return, which is a reversed mapping.

Comment (samuelhwilliams) [inline pages_builder/pages/forms/selection-buttons.yml]: ah ok

File: toolkit/templates/forms/list-entry.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 {% elif message %}
   <div class="message-wrapper">
 {% endif %}
-  <fieldset class="question {% if first_question %}first-question{% endif %}" id="{{ id }}">
+  <fieldset class="question {% if first_question %}first-question{% endif %}{% if hidden %} js-hidden related-information{% endif %}" id="{{ id }}">
```
Comment (pcraig3) [inline toolkit/templates/forms/list-entry.html]: This is a thing we've done in a few places where we add extra spaces to classnames. 
If you remove the space after `class="question ` and put it before `first-question`, it's more correct.

File: toolkit/templates/forms/selection-buttons.html [hunk 3]
```diff
@@ -45,67 +45,41 @@
...
+    {% endif %}
+
+    {% if type in ["radio", "boolean"] %}
+      {% set input_type = "radio" %}
+    {% else %}
+      {% set input_type = "checkbox" %}
+    {% endif %}
+
+    {% for option in options %}
+      {% set input_value = option.value if 'value' in option else option.label %}
```
Comment (pcraig3) [inline toolkit/templates/forms/selection-buttons.html]: Good call fixing this stuff.  I know @benvand was asking about it.

**Comment by pcraig3** [comment]
I think go for it. 👍

**Pull request merged**